blob/example_test.go

```diff
@@ -148,7 +148,7 @@ func Example() {
 
 	// We now have a *blob.Bucket! We can write our application using the
 	// *blob.Bucket type, and have the freedom to change the initialization code
-	// above to choose a different provider later.
+	// above to choose a different driver later.
```

Comment:
service-specific driver
(since this is an educational example, it might help to be more descriptive)

Reply:
Done.